### PR TITLE
{bio}[iccifort/2019.1.144-GCC-8.2.0-2.31.1] BamTools v2.5.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3126.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/14c91b76a5f668286a900fef16ef52be for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2099.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e22e3be1c12bca0478904171bb6f1633 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2658.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/137bed3886e0e26bf73b39883a3ce152 for a full test report.

File: easybuild/easyconfigs/b/BamTools/BamTools-2.5.1-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+description = "BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files."
+
+toolchain = {'name': 'iccifort', 'version': '2019.1.144-GCC-8.2.0-2.31.1'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/pezmaster31/%(namelower)s/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['4abd76cbe1ca89d51abc26bf43a92359e5677f34a8258b901a01f38c897873fc']
+
+builddependencies = [('CMake', '3.13.3')]
```
Comment (akesandgren) [inline easybuild/easyconfigs/b/BamTools/BamTools-2.5.1-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb]: The foss-2017b version has pkg-config as a build dep, isn't that needed?

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/42cb9aedf60a3e5f29ef7c48dcf97428 for a full test report.

File: easybuild/easyconfigs/b/BamTools/BamTools-2.5.1-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb (hunk 1 above)
Comment (vanzod) [reply easybuild/easyconfigs/b/BamTools/BamTools-2.5.1-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb]: @boegel `pkg-config` is still required. I will push a PR that will add the dependency to the existing easyconfig files that require it.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/94801ad7d062b913e703f63dc5240f3a for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Pull request merged**